src/node_sqlite.cc

```diff
@@ -506,6 +508,194 @@ void StatementSync::All(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(Array::New(isolate, rows.data(), rows.size()));
 }
 
+void StatementSync::IterateReturnCallback(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  auto isolate = env->isolate();
+  auto context = isolate->GetCurrentContext();
+
+  auto self = args.This();
+  // iterator has fetch all result or break, prevent next func to return result
+  self->Set(context,
+            String::NewFromUtf8Literal(isolate, "is_finished"),
+            Boolean::New(isolate, true))
+      .ToChecked();
```

Comment:
These should be `env` strings... e.g. `env->is_finished_string()` , etc ... here and below.

Reply:
Done, no `String::NewFromUtf8Literal` in file